test/web-auth/web-auth.test.js

```diff
@@ -2402,7 +2404,7 @@ describe('auth0.WebAuth', function() {
       stub(IframeHandler.prototype, 'init', function() {
         this.timeoutCallback();
       });
-      this.auth0.checkSession({}, function(err, data) {
+      this.auth0.checkSession({ state: 'foobar' }, function(err, data) {
```

Comment:
why has this changed? If I read the tests below (line 2452) it seems it's using this key as default state key value to remove when no error is provided (because of timeout). What would happen if this value is not given, like in the previous diff, when an error is also not provided (because of timeout)?

Reply:
Because the underlying code now depends on a state and my `stub` for the `transactionManager.process` method doesn't return a state: https://github.com/auth0/auth0.js/blob/6ac59c4faedc003820f3909aef5b5d57a572fad7/test/web-auth/web-auth.test.js#L2303

Since I already tested (elsewhere)[https://github.com/auth0/auth0.js/blob/6ac59c4faedc003820f3909aef5b5d57a572fad7/test/web-auth/web-auth.test.js#L2362] that we handle those params correctly, there's no need to stub the method again, so I just added the state